src/main/java/seedu/address/logic/parser/quiz/QuizAnswerParser.java

```diff
@@ -33,7 +33,7 @@ public QuizCheckAnswer parse(String userInput) throws ParseException {
         final String index = matcher.group("index");
         final String stringAnswer = matcher.group("answer");
         final int intIndex = Integer.parseInt(index) - 1;
-        final Answer answer = new Answer(stringAnswer);
+        final Answer answer = new Answer(stringAnswer.trim());
```

Comment:
You should move the `trim()` method call to the place where `stringAnswer` is defined, i.e. `matcher.group("answer").trim()`, rather than call the two chainable methods on two different lines.